dnf-behave-tests/features/plugins-core/repomanage.feature

```diff
@@ -0,0 +1,202 @@
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/src/nodejs-5.3.1-1.module_3012+41787ba3.src.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/src/nodejs-5.4.1-2.module_2011+41787af1.src.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/src/nodejs-5.4.1-2.module_3012+41787ba4.src.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/src/postgresql-9.6.8-1.module_1710+b535a823.src.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/nodejs-5.12.1-1.fc29.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/nodejs-5.4.1-2.module_2011+41787af1.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/nodejs-5.4.1-2.module_3012+41787ba4.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/nodejs-devel-5.12.1-1.fc29.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/npm-5.12.1-1.fc29.x86_64.rpm
+      """
+
+
+@bz1804720
+Scenario: only the newest modular packages get listed as new
+Given I copy repository "dnf-ci-multicontext-hybrid-multiversion-modular" for modification
+ When I execute dnf with args "repomanage --new {context.dnf.repos[dnf-ci-multicontext-hybrid-multiversion-modular].path}"
+ Then the exit code is 0
+  And stdout is
+      """
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/noarch/nodejs-docs-5.12.2-3.fc29.noarch.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/src/nodejs-5.12.2-3.fc29.src.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/src/postgresql-9.8.1-1.module_9790+c535b823.src.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/nodejs-5.12.2-3.fc29.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/nodejs-5.3.1-1.module_2011+41787af0.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/nodejs-5.3.1-1.module_3012+41787ba3.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/nodejs-devel-5.12.2-3.fc29.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/npm-5.12.2-3.fc29.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/postgresql-9.6.8-1.module_1710+b535a823.x86_64.rpm
+      {context.dnf.tempdir}/repos/dnf-ci-multicontext-hybrid-multiversion-modular/x86_64/postgresql-9.8.1-1.module_9790+c535b823.x86_64.rpm
+      """
+
+
+@bz1804720
+Scenario: all packages get listed as new if --keep is really big
```

Comment:
This is just a nitpicking, but can you please substitute "really" for "sufficiently"? Because "really" is really vague and kind of sounds like there is some magic number above which all packages get listed.